app/assets/stylesheets/typography.scss

```diff
@@ -0,0 +1,49 @@
+.display {
+  font-size: 36px;
+  font-weight: 300;
+  letter-spacing: 0.4;
+  color: $text-primary;
+  font-family: $headline-fonts;
+}
+.headline {
+  font-size: 24px;
+  letter-spacing: 0.3;
+  color: $text-primary;
+  font-family: $headline-fonts;
+}
+.title {
+  font-size: 20px;
+  letter-spacing: 0.2;
+  color: $text-primary;
+  font-family: $headline-fonts;
+}
+.subheader {
```

Comment:
`.subheader`, `.title`, `.headline`, `.display` all use the same `color` and `font-family`. So, why repeat the definitions?

Can we instead have a single definition for each of those properties?